GitUI/GitUICommands.cs

```diff
@@ -1049,7 +977,7 @@ public bool StartRemotesDialog(IWin32Window owner, string preselectRemote = null
         {
             bool Action()
             {
-                using (var form = new FormRemotes(this))
+                using var form = new FormRemotes(this);
                 {
```

Comment:
Either `{ }` or `;` is incorrect here.

Reply:
This was a manual change...
Will squash after drew is done